config/trackdirect.ini

```diff
@@ -5,9 +5,9 @@
 [database]
 
 ;; Basic database settings
-host="127.0.0.1"
+host="db"
 database="trackdirect"
-;username="username"
+username="root"
```

Comment:
By not setting a username the current username is used, and that is usually what you want (if not using docker)

Let's write a comment here:
`;; If using docker, set username to "root"`